src/Ionide.ProjInfo/Library.fs

```diff
@@ -331,10 +331,10 @@ module ProjectLoader =
 
     type LoadedProject = internal LoadedProject of ProjectInstance
 
-    let internal msBuildLogger = lazy (LogProvider.getLoggerByName "MsBuild") //lazy because dotnet test wont pickup our logger otherwise
+    let internal projectLoaderLogger = LogProvider.getLoggerByName "ProjectLoader"
```

Comment:
I've hit some weird edges based on how module scoped variables are loaded statically in F#, and you might be stuck with a `Serilog.SilentLogger` if it's not lazy/thunked. If it seems fine I have no idea why I hit this in the first place.

Reply:
good spot - wrapped and pushed